### DISCUSSION TITLE
Created org using an api but the org was there in cassandra but not in the ES

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Lern/Community#113] [Issues] [answered]
Created org using an api but the org was there in cassandra but not in the ES when we do the org read it was working the API is 

`curl --location 'https://dev.sunbirded.org/api/org/v1/read' \
--header 'Accept: application/json' \
--header 'Content-Type: application/json' \
--header 'X-Authenticated-User-Token: <X-Authenticated-User-Token>' \
--header 'Authorization: Bearer <Authorization>' \
--data '{
  "request": {
    "organisationId": "0137541424673095687"
  }
}'`
But for the same org the org search API was not working
`curl --location 'https://dev.sunbirded.org/api/org/v1/search' \
--header 'Accept: application/json' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer <Authorization>' \
--data '{
  "request": {
    "filters": {
        "identifier": "0137541424673095687"
    }
  }
}'`

as the org was not available in the Elastic search 
then we had to sync using the data sync API
`curl --location 'https://dev.sunbirded.org//api/data/v1/index/sync' \
--header 'Authorization: Bearer <Authorization>' \
--header 'Content-Type: application/json' \
--data '{
  "params": {},
  "request": {
    "objectType": "organisation",
    "objectIds": [
      "0137541424673095687"
    ]
  }
}'`

cc: @reshmi-nair @vinukumar-vs

**Comment by reshmi-nair** [comment]
@rajeevsathish There might have been some problem with elastic search during org creation. If all the setup is correct this does not happen. In case if it happens again, please share the server logs..

**Reply by vinukumar-vs** [reply]
@reshmi-nair Is there any way to verify in the logs for failure? Today, it's very difficult to what happen & how to fix. 

These can be added to your FAQ section which help for other adopters.

**Reply by reshmi-nair** [reply, marked as answer]
@vinukumar-vs Learner pod logs need to be checked for this error : "saveDataToES: Exception occurred" , along with this stacktrace will be there. Also after creating organisation, please use the org search API and validate if it actually added. Will create a troubleshooting section in microsite and update these steps.
